oqs_scripts/testrun.py

```diff
@@ -0,0 +1,38 @@
+import json
```

Comment:
Like `oqs_try_handshake.py`, can this be placed in the project root and renamed to something like `oqs_test_with_interop_server.py`?

Reply:
scripts in a project main folder are not good - so followed your statement below

> We can also rename oqs_try_handshake.py to oqs_scripts/try_handshake.py